transform/models/marts/geo_reference/dim_geo_reference_footprints_blocks__joined.sql

```diff
@@ -0,0 +1,42 @@
+with footprints as (
+    select
+        "release",
+        "capture_dates_range",
+        "geometry"
+    from {{ ref('stg_building__footprints') }}
+),
+
+blocks as (
+    select
+        "county_fp",
+        "tract_ce",
+        "block_ce",
+        "geo_id",
+        "name",
+        "maf_tiger_feature_class_code",
+        "maf_tiger_feature_class",
+        "urban_rural_area",
+        "housing_unit_count",
+        "pop_count",
+        "geometry"
+    from {{ ref('int_geo_reference_blocks__mapped') }}
+),
+
+footprints_and_blocks_joined as (
+    select
+        footprints.*,
+        blocks."county_fp",
+        blocks."tract_ce",
+        blocks."block_ce",
+        blocks."geo_id",
+        blocks."name",
+        blocks."maf_tiger_feature_class_code",
+        blocks."maf_tiger_feature_class",
+        blocks."urban_rural_area",
+        blocks."housing_unit_count",
+        blocks."pop_count"
```

Comment:
This is perfectly fine! But it's also an opportunity for the new-ish Snowflake [EXCLUDE](https://docs.snowflake.com/en/sql-reference/sql/select) syntax

I'm a bit concerned that this table is getting a bit wide. To me, the intention here is for the table to be easily filtered and joined with other census data. In that case, I'm not sure that we need things like `maf_tiger_feature_class_code` or `housing_unit_count`.

Reply:
Thinking about it a bit more: I think we should drop the demographic-related things and just stick to block, tract, etc identifiers here. Things like `pop_count` are kind of confusing here. It looks like it's attached to the building footprint, but maybe it's for the tract, or for the county? What is urban/rural referring to? *We* know that the answer is referring to the block, but it's not clear from a glance at the table.

Reply:
Is the intent to make the table useful for glancing or useful for gathering related dimensions. Can you elaborate on why you think the `maf_tiger_feature_class_code`, `pop_count`, or `housing_unit_count` dimensions are not useful?

Reply:
I think they have the potential to be misleading: the effective "primary key" for this table is the building footprint. `pop_count` looks like it might refer to the building. So might `housing_unit_count` (what would that mean?). Or whether it's urban or rural.

As the implementer, you know that it's referring to the census *block*, but the casual user of the dataset might not. I think the intended use of this table is to provide *census-related-GEOIDs*, which make it easy to join on any other census data. This goes way beyond `pop_count`, or `housing_unit_count`.

Reply:
This clarification is helpful, I agree. Thank you!